mmdet3d/datasets/kitti_dataset.py

```diff
@@ -86,7 +86,7 @@ def parse_data_info(self, info: dict) -> dict:
 
         Returns:
             dict: Has `ann_info` in training stage. And
-            all path has been converted to absolute path.
+                all path has been converted to absolute path.
```

Comment:
the original docstr format is correct
The format of `return` is different from the args